src/extensionsIntegrated/TabBar/main.js

```diff
@@ -128,23 +122,42 @@ define(function (require, exports, module) {
         const activePathInPane = activeFileInPane ? activeFileInPane.fullPath : null;
 
         // Check if this file is active in its pane
-        const isActive = (entry.path === activePathInPane);
+        const isActive = entry.path === activePathInPane;
 
         // Current active pane (used to determine whether to add the blue underline)
         const currentActivePane = MainViewManager.getActivePaneId();
-        const isPaneActive = (paneId === currentActivePane);
+        const isPaneActive = paneId === currentActivePane;
```

Comment:
`const isPaneActive = (paneId === currentActivePane);` this with explicit braces is preferred over implicit to be more readable and prevent operator precedence confusions. i think this came as part of a whole file beautify. Just FYI, no need to update this pr.